docs/en/sql-reference/functions/splitting-merging-functions.md

```diff
@@ -38,6 +38,8 @@ The behavior of parameter `max_substrings` changed starting with ClickHouse v22.
 For example,
 - in v22.10: `SELECT splitByChar('=', 'a=b=c=d', 2); -- ['a','b','c=d']`
 - in v22.11: `SELECT splitByChar('=', 'a=b=c=d', 2); -- ['a','b']`
+
+The previous behavior can be restored by setting [splitby_max_substring_behavior](../../operations/settings/settings.md#splitby-max-substring-behavior) = 'python'.
```

Comment:
we can probably use the compatibility setting to set it to correct behaviour and mention here that it can be used also.

Reply:
Will check.

Reply:
The new behavior with `splitby_max_substrings_includes_remaining_string = 1` doesn't match 100% the pre-v22.11 behavior (one needs to adjust `max_substrings` argument by 1). But I feel that's an acceptable compromise which reduces implementation complexity and understanding for users. So I better refrain from compatibility settings, to keep surprises minimal.